src/main/kotlin/io/arct/arctlib/plugin/commands/Command.kt

```diff
@@ -28,9 +29,12 @@ abstract class Command(val name: String) : CommandExecutor {
         for (permission in permissions)
             if (!sender.hasPermission(permission)) {
                 plugin raise PermissionException() send sender
-                return true
+                return false
```

Comment:
Please revert this change. Returning false will print usage information which is not desired.